Use system zlib library instead of embedded lzma in courgette.

Author: Michael Gilbert <mgilbert@debian.org>
Origin: https://salsa.debian.org/chromium-team/chromium/raw/6ecd074/debian/patches/system/zlib.patch

--- a/courgette/BUILD.gn
+++ b/courgette/BUILD.gn
@@ -60,7 +60,6 @@ static_library("courgette_lib") {
 
   deps = [
     "//base",
-    "//third_party/lzma_sdk",
   ]
 
   public_deps = [
@@ -81,7 +80,6 @@ static_library("courgette_lib") {
   ]
   deps = [
     "//base",
-    "//third_party/lzma_sdk",
   ]
 }
 
--- a/courgette/crc.cc
+++ b/courgette/crc.cc
@@ -7,6 +7,8 @@
 #include <stdint.h>
 #include <stddef.h>
 
+#define COURGETTE_USE_CRC_LIB
+
 #ifdef COURGETTE_USE_CRC_LIB
 #  include "zlib.h"
 #else
